### PR TITLE
Previous version entry

### PR DESCRIPTION
**Pull request by jdrichards50**

### CODE REVIEW

**Comment by jeckenrode-ciq** [comment]
I think "For previous version support, please refer to the Releases section." should be the first line under "Version 8.2.1 or later of the ChartIQ library" in Requirements.

Then, I think the names of the releases should indicate ChartIQ library versions rather than app versions. Maybe something like: "ChartIQ-Android-SDK for v8.1.0" or "Android SDK for ChartIQ v8.1.0".

**Comment by jdrichards50** [comment]
> I think "For previous version support, please refer to the Releases section." should be the first line under "Version 8.2.1 or later of the ChartIQ library" in Requirements.
> 
> Then, I think the names of the releases should indicate ChartIQ library versions rather than app versions. Maybe something like: "ChartIQ-Android-SDK for v8.1.0" or "Android SDK for ChartIQ v8.1.0".

We definitely can't add that to the tag name. It also doesn't make sense. The app need their own releases. It looks like I can add that to the tag description maybe.

**Comment by jeckenrode-ciq** [comment]
This looks good, Jacob. I wasn't able to associate the app versions with library versions. The comments fix that.

The only other thing is the "For previous version support . . ." needs to be indented two spaces so that it lines up with "Version 8.2.1 or later . . ."

File: README.md [hunk 1]
```diff
@@ -10,6 +10,8 @@ Contact us at <support@chartiq.com> to request sample code and guidance on how t
 
 - Version 8.2.1 or later of the ChartIQ library
 
+For previous version support, please refer to the [Releases](https://github.com/ChartIQ/ChartIQ-Android-SDK/releases) section.
```
Comment (jeckenrode-ciq) [inline README.md:13]:
```suggestion
  For previous version support, please refer to the [Releases](https://github.com/ChartIQ/ChartIQ-Android-SDK/releases) section.
```

**Review by jeckenrode-ciq — APPROVED** [review]
Looks good.

**Pull request merged**